IFComp/root/src/admin/feedback/index.tt

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+
+<h1>All feedback</h1>
+
+<p>
+Here is all the feedback received this year, sorted by judge name.</p>
+
+<p><strong>Please consider the contents of this page sensitive information.</strong> Outside of this page, only the author of a given entry can see the feedback left for it, and in that case they cannot see the identity of the judge who wrote it (unless the judge volunteered this information in the text of the feedback itself.)
+</p>
+
+<hr />
+
+[% IF feedback_rs.count %]
+    [% WHILE (feedback = feedback_rs.next) %]
+        <p><strong>[% feedback.judge.name %] ([% feedback.judge.email %])</strong>, on [% feedback.entry.title %]:</p>
```

Comment:
Why aren't the name, email, and title being passed through the `html` filter?

Reply:
I feel comfortable saying that if these fields are allowed to contain HTML-unsafe characters then we have bigger problems. And maybe we do! But if so they should be addressed as their own issue, and then tackled at the form-validation level.

Reply:
You wouldn't allow an ampersand in an entry title?

The title and author are already passed through the `html` filter on the ballot page. There's no reason not to do it here, too, even if you don't expect it to be required. (In fact, why not re-use the existing `_entry_title.tt` and `author_name.tt` templates?)

Reply:
You're right, was confusing the HTML filter for tag-stripping.